utils/dataset_manifest/core.py

```diff
@@ -706,12 +706,15 @@ def _validate_first_item(_dict):
             raise ValueError('Incorrect name field')
         if not isinstance(_dict['extension'], str):
             raise ValueError('Incorrect extension field')
-        # width and height are required for 2d data
-        # FIXME for 3d when manual preparation of the manifest will be implemented
-        if not isinstance(_dict['width'], int):
-            raise ValueError('Incorrect width field')
-        if not isinstance(_dict['height'], int):
-            raise ValueError('Incorrect height field')
+        # FIXME
```

Comment:
@Marishka17 , the same question. What are we going to do with the comment? It looks like we disable two checks which can be important for 2D case.

Reply:
@nmanovic,  yes, but these checks are only relevant for the first element of the manifest file, and the reliability will drop a bit...